modules/howtos/pages/managing-connections.adoc

```diff
@@ -55,6 +55,10 @@ include::example$managing_connections.py[tag=customenv,indent=0]
 [#connection-strings]
 == Connection Strings
 
+WARNING: The backend implementation of connection strings parameters changed substantially in 4.0 and is not currently fully documented.
+This will be resolved in a future 4.x release.
+See xref:project-docs:migrating-sdk-code-to-3.n.adoc#sdk4-specifics[more details on migrating to 4.0].
+
 A Couchbase connection string is a comma-delimited list of IP addresses and/or hostnames, optionally followed by a list of parameters.
 
 The parameter list is just like the query component of a URI; name-value pairs have an equals sign (`=`) separating the name and value, with an ampersand (`&`) between each pair.
```

Comment:
Can we mention that while this is possible, it is preferred to pass in options via ClusterOptions()?